### DISCUSSION TITLE
Static action method "message" for queued actions.

### DISCUSSION THREAD

**Discussion by HenriqueSPin** [laravel/nova-issues#4592] [Ideas & Feature Requests] [answered]
It would be nice if Nova checks for a static method called "message" in a queue action class that would allow us to customize the feedback message shown to the user.

Example: "Your report is being generated. You will be notified."

**Comment by davidhemphill** [comment, marked as answer]
I've added an update to allow for this. You need to use the `handleResult` method on your action instance. Nova will use the last result for the custom message:

```php
<?php

namespace App\Nova\Actions;

//...

class QueuedAction extends Action implements ShouldQueue
	//...

    public function handleResult(ActionFields $fields, $results)
    {
        return Action::message('The action has been queued.');
    }
}
```

**Reply by HenriqueSPin** [reply]
Worked perfectly! Thank you.
